components/AppLayout/AppLayout.js

```diff
@@ -97,6 +99,7 @@ function AppLayout({ children, container = true }) {
     };
     const handleRouteChangeComplete = () => {
       setRouteChanging(false);
+      window.scrollTo(0, 0);
```

Comment:
I think the browser already handles scroll position across navigation.
Why should we reset scroll here?